### PR TITLE
master2main and dev env setup

### PR DESCRIPTION
**Pull request by TanyaStere42**
changes:
- changed all references to wb:master
- included dev setup instructions
- updated pr workflow to references dev setup

### CODE REVIEW

File: docs/contribute/dev-setup.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+1. verify the war file was created in `target/wildbook-X.Y.Z.war` (where X.Y.Z is the current version number).
+1. cd to the deploy directory `cd ~/wildbook-dev/webapps/wildbook` 
+1. deploy your warfile `jar -xvf /code/directory/Wildbook/target/wildbook-X.Y.Z.war`
+
+### Deploy
+1. `cd` to the `devops/development` directory in the code repo
+1. run `docker-compose up [-d]`
+1. To verify successful launch, open in browser `http://localhost:81/` when tomcat has started. Default login of username/password `tomcat`/`tomcat123` should work.
+
+Note: if you're running docker as root, you may way to explicitly set your deploy directory path to include your user, i.e., `WILDBOOK_BASE_DIR=~USER/wildbook-dev`
```
Comment (goddesswarship) [inline docs/contribute/dev-setup.md]: Typo: "you may way to explicitly..." should likely be "you may want to explicitly..."

**Review by goddesswarship — COMMENTED** [review]
Found one typo in line 53, but otherwise all good!

File: docs/contribute/dev-setup.md (hunk 1 above)
Comment (TanyaStere42) [reply docs/contribute/dev-setup.md]: addressed, good catch

**Review by goddesswarship — APPROVED** [review]
🦖

**Pull request merged**